site/src/docs/foundation/visual-assets/wave-motifs.mdx

```diff
@@ -25,8 +25,8 @@ Wave motifs are available in six (6) different shapes:
 - Basic (<span lang="fi">Perus</span>)
 - Pulse (<span lang="fi">Pulssi</span>)
 - Beat (<span lang="fi">Syke</span>)
-- Storm (<span lang="fi">Tyrsky</span>)
-- Wave (<span lang="fi">Värinä</span>)
+- Wave (<span lang="fi">Tyrsky</span>)
```

Comment:
Good find! This was discussed in Slack (and maybe in another icon PR, i don't remember) that we are waiting for finnish translations. So this can be fixed later.